lib/roo/xls/excel_2003_xml.rb

```diff
@@ -18,8 +18,10 @@ def initialize(filename, options = {})
       unless File.file?(@filename)
         fail IOError, "file #{@filename} does not exist"
       end
-      @doc = load_xml(@filename)
+      @doc = ::Roo::Utils.load_xml(@filename)
     end
+    namespace = @doc.namespaces.select{|xmlns, urn| urn == 'urn:schemas-microsoft-com:office:spreadsheet'}.keys.last
+    @namespace = namespace.nil? || namespace.empty? ? 'ss' : namespace.split(':').last
```

Comment:
I like it. Although it not self-explaining anymore. One could then turn it into a one-liner (since `namesapce` is not used elsewhere).
```
@namespace =  (@doc.namespaces.key('urn:schemas-microsoft-com:office:spreadsheet') || 'ss').split(':').last
```

What do you think?